src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issueadjournment/IssueAdjournmentNoticeAboutToStartHandler.java

```diff
@@ -44,10 +45,14 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
 
         PreSubmitCallbackResponse<SscsCaseData> response = new PreSubmitCallbackResponse<>(sscsCaseData);
 
-        if (sscsCaseData.getAdjournCasePreviewDocument() != null) {
+        // Give the user selects No to generate notice option and has uploaded their custom adjournment notice document.
+        // Then don't generate the system default adjournment notice document.
+        if (sscsCaseData.getAdjournCaseGenerateNotice().equalsIgnoreCase(YesNo.YES.getValue())) {
```

Comment:
When doing a .equals check you should always do that on the constant not the variable. The constant will never be null but the variable may be. If you do that you avoid a null exception.
In this case you should use the YesNo.isYes(...) method, which is the way we test for Yes and No's.